SimonSaysLab/SimonSaysGame.swift

```diff
@@ -95,7 +112,4 @@ extension SimonSays {
         chosenColors.removeLast()
```

Comment:
Might be too nit-picky here, but I think we can mark this `makeGuessWith()` function private, that way the student doesn't try to call on it from the ViewController.swift file.